addons/mail/static/src/models/message/message.js

```diff
@@ -112,7 +112,14 @@ function factory(dependencies) {
                 }
             }
             if ('partner_ids' in data && this.messaging.currentPartner) {
-                data2.isCurrentPartnerMentioned = data.partner_ids.includes(this.messaging.currentPartner.id);
+                data2.recipients = insertAndReplace(data.recipients.map(recipient =>
+                    this.messaging.models['mail.partner'].convertData(recipient)
+                ));
+            }
```

Comment:
Isn't partner_ids just a list of ids? You'd need to convert it to { id: value } object before insert.

Reply:
Try to temporarily remove `recipients` key in python to see how it would react.